### DISCUSSION TITLE
Can I use Raspberry Pi 3 B instead of B+?

### DISCUSSION THREAD

**Discussion by Solenepradat** [librespot-org/librespot#810] [General]
Hi everyone, 

I'm super new to this field so I might be asking quite a dumb question, but better safe than sorry :) 
Can I use a raspberry Pi 3b instead of raspberry Pi 3b+ to build this phoniebox? The list of materials says 3b+, which is why I'm asking... 

Thanks for your help!

**Comment by roderickvd** [comment]
I don't know what you mean by "phoniebox" but if you intend to compile `librespot` on your own then sure. The lowest spec we are targeting is a RPi 0. If you're entirely new then you might want to look at some prepackaged editions like [Raspotify](https://github.com/dtcooper/raspotify) that does the heavy lifting for you.
